conf/messages.en

```diff
@@ -215,6 +216,7 @@ movement.status.unloadingRemarksSubmitted = Unloading remarks sent
 movement.status.unloadingRemarksRejected = Unloading remarks rejected
 movement.status.UnloadingRemarksXMLSubmissionNegativeAcknowledgement = Unloading remarks rejected
 movement.status.goodsReleased = Goods Released
+movement.status.P5.arrival.goodsNotReleased= Goods not released - waiting for discrepancy resolution
```

Comment:
Not sure if this is a duplicate got slightly different message keys for the same message on line 152

Reply:
152 UPDATED AS THAT IS USED IN DEPARTURES ONLY